.github/workflows/linux.yml

```diff
@@ -28,20 +28,21 @@ jobs:
     - name: Basic build
       run: go build ./cmd/...
 
-    - name: Test build
-      run: go test ./...
+    - name: macOS build
+      env:
+        GOOS: darwin
+        GOARCH: amd64
+      run: go build ./cmd/...
 
-    - uses: k0kubun/action-slack@v2.0.0
```

Comment:
Why is this being deleted?

Reply:
Temporary, to not spam our slack.

#42 should be fixed now, so a rebase + rerun should do the trick.

Reply:
Looks green all around now. I've added this back in.